src/main/java/com/google/sps/data/Election.java

```diff
@@ -140,6 +199,8 @@ public boolean isPopulatedByVoterInfoQuery() {
   public static Election fromEntity(Entity entity) {
     Set<Long> contests = new HashSet<>();
     Set<Long> propositions = new HashSet<>();
+    ArrayList<EmbeddedEntity> pollingStations = new ArrayList<EmbeddedEntity>();
```

Comment:
List<EmbeddedEntity> pollingStations = new ArrayList<>();

(1) Java is smart enough to infer the type at runtime. 
(2) Prefer the more general interface.

Reply:
Doing so gives me the following error:

java.util.List<com.google.appengine.api.datastore.EmbeddedEntity> cannot be converted to java.util.ArrayList<com.google.appengine.api.datastore.EmbeddedEntity>

Reply:
I think that's because you have to change the builder function to also take in a list. And return a list. 

But anyway, combined with the comment from above about immutableset, all this should be converted to sets/immutableset. 

Here:
Set<EmbeddedEntity> pollingStations = new HashSet<>();

In builder:
public abstract Builder setPollingStations(Set<EmbeddedEntity> pollingStations);

In autovalue:
public abstract ImmutableSet<EmbeddedEntity> pollingStations();
----------
The same advice from before applies though. 
 * Set<EmbeddedEntity> in the variable declaration because you want it more generic. 
 * new HashSet<> in the instantiation because Java is smart enough to infer the type
 * ImmutableSet because mutability is bad